wcfsetup/install/files/lib/system/condition/UserStateCondition.class.php

```diff
@@ -47,6 +47,19 @@ class UserStateCondition extends AbstractSingleFieldCondition implements IConten
 	 */
 	protected $userIsNotBanned = 0;
 	
+	/**
+	 * true if the the user is be email confirmed
+	 *
+	 * @var	integer
+	 */
+	protected $userIsEmailConfirmed = 0;
+	
+	/**
+	 * true if the the user is not be email confirmed
```

Comment:
Sounds a little weird. Perhaps `true if the the user has not confirmed their email` (address?)